### PR TITLE
remove lwt support from serializers and access, bandwagon

### LINKED ISSUE #11703: remove lwt support in serializers
We have specific support to return `theme_data` for LWT but with no LWT any more we can delete that code now.

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #11703

### CODE REVIEW

File: src/olympia/addons/indexers.py [hunk 1]
```diff
@@ -325,44 +323,15 @@ def extract_document(cls, obj):
...
-        else:
-            if obj.current_version:
-                data['platforms'] = [p.id for p in
-                                     obj.current_version.supported_platforms]
-
-            # Extract dominant colors from static themes.
-            if obj.type == amo.ADDON_STATICTHEME:
-                first_preview = obj.current_previews.first()
-                if first_preview:
-                    data['colors'] = first_preview.colors
```
Comment (EnTeQuAk) [inline src/olympia/addons/indexers.py]: This needs a re-index, mind making sure it get's added to next weeks deployment milestone?

File: src/olympia/bandwagon/models.py [hunk 2]
```diff
@@ -93,10 +93,6 @@ class Collection(ModelBase):
     addon_count = models.PositiveIntegerField(default=0,
                                               db_column='addonCount')
 
-    all_personas = models.BooleanField(
-        default=False,
-        help_text='Does this collection only contain Themes?')
```
Comment (EnTeQuAk) [inline src/olympia/bandwagon/models.py]: This needs a database migration, mind filing an issue for that?

File: src/olympia/migrations/600-theme-description.py [hunk 3]
```diff
@@ -1,18 +1,14 @@
...
-        summary__isnull=False)
-    for addon in addons:
-        addon.description = addon.summary
-        addon.save()
+    # addons = Addon.objects.filter(
+    #     type=amo.ADDON_PERSONA, description__isnull=True,
+    #     summary__isnull=False)
+    # for addon in addons:
+    #     addon.description = addon.summary
+    #     addon.save()
```
Comment (EnTeQuAk) [inline src/olympia/migrations/600-theme-description.py]: :roll_eyes:

Comment (eviljeff) [reply src/olympia/migrations/600-theme-description.py]: so... just delete it?  the other migration I found commented it out so I did the same.

File: src/olympia/bandwagon/models.py (hunk 2 above)
Comment (eviljeff) [reply src/olympia/bandwagon/models.py]: #11718

File: src/olympia/addons/indexers.py (hunk 1 above)
Comment (eviljeff) [reply src/olympia/addons/indexers.py]: https://github.com/mozilla/addons/blob/master/releases/2019/07/04.md

**Pull request merged**